database_setup/aggregated-user-tracking.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE public.user_tracking
```

Comment:
Shall we name the table `user` so it's more generic as this will probably contain user preferences as well